### DISCUSSION TITLE
Usable to monitor countdown of washing maschine ?

### DISCUSSION THREAD

**Discussion by aalex2014** [jomjol/AI-on-the-edge-device#2141] [Ideas]
Hello.
I have the idea to monitor our washing machine (and our dryer) which are both not smart home ready. They have a minute countdown on their display. And I would like to receive a notification when they are done.
So, simple question. Can AI-on-the-edge-device be used for monitoring such a thing? The difference here is that it's not a constantly increasing counter, but a temporary visible countdown timer counting hours and minutes.

Thanks for your thoughts. -Alex

**Comment by ThePhantom79** [comment]
You can recognize these digits and send them to Home Assistant / MQTT , thats not the problem. 
But the analysis of the digits (now its "0:00", then do this or that) and the notification you have to be externally (by home assistant for example. That is not part of this project. It's only to "read" / recognize the digits and send them by MQTT ...
